### PR TITLE
Now building RPMs for Centos 6!

### PR DESCRIPTION
**Pull request by mathuin**
https://github.com/alanfranz/fpm-within-docker is an awesome thing.

The example project was easily tweakable for our needs.

If accepted, this will close issues #1 and #2!

### CODE REVIEW

File: packaging/centos-6/build-image/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,33 @@
+FROM alanfranz/fwd-centos-6:latest
```
Comment (ramereth) [inline packaging/centos-6/build-image/Dockerfile]: Can you please build this for CentOS 7 instead?

File: packaging/centos-6/build-image/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    tar \
+    wget \
+    zlib \
+    zlib-devel \
+    zlib-static \
+    && yum clean all
+RUN mkdir /dssp
+WORKDIR /dssp
+RUN wget http://www.bzip.org/1.0.6/bzip2-1.0.6.tar.gz && \
+    tar zxf bzip2-1.0.6.tar.gz
```
Comment (ramereth) [inline packaging/centos-6/build-image/Dockerfile]: Why do you need to install bzip2 manually?

File: packaging/centos-6/build-image/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+WORKDIR /dssp/bzip2-1.0.6
+RUN mv Makefile Makefile.old && sed -e "/^CFLAGS/s/$/ -fPIC/" < Makefile.old > Makefile && make
+RUN make install
+WORKDIR /dssp
+RUN rm -rf bzip2-1.0.6 bzip2-1.0.6.tar.gz
+RUN wget http://downloads.sourceforge.net/project/boost/boost/1.57.0/boost_1_57_0.tar.gz && \
+    tar zxf boost_1_57_0.tar.gz
```
Comment (ramereth) [inline packaging/centos-6/build-image/Dockerfile]: Why do you need to install boost manually?

File: packaging/centos-6/build-image/Dockerfile (hunk 2 above)
Comment (mathuin) [inline packaging/centos-6/build-image/Dockerfile]: mkdssp, the binary we're making, is statically linked.  It requires static libraries from bzip2 and boost.  The Centos packages for both of these products only include dynamic libraries, not static ones.

In the future, a pull request to convert mkdssp to dynamic linking would make the installation of bzip2 and boost unnecessary at this point but would them require the addition of dependencies on the bzip2 and boost packages.

File: packaging/centos-6/build-image/Dockerfile (hunk 3 above)
Comment (mathuin) [inline packaging/centos-6/build-image/Dockerfile]: Answered this one with bzip2 -- I need the static libraries which aren't in the packages.

**Comment by mathuin** [comment]
Okay, I just changed this to build for Centos 7.  I will make another pull request instead of this one, because I accidentally told it to push to master instead of develop.

**Pull request closed without merging**